src/main/java/pl/nogacz/snake/board/Board.java

```diff
@@ -85,8 +91,23 @@ private void moveSnake() {
 
     private void moveSnakeHead(Coordinates coordinates) {
         if(coordinates.isValid()) {
+            if(!isRottenFoodExist()) {

```

Comment:
Bu bölgeyi bir metota al

Reply:
Evet yüzdeyi belirtiyor, yorum satırı eklemeye çalıştım yeterli değilse nasıl yapmamı tavsiye edersiniz?

Reply:
Son commit ile yeni bir değişkende ekleme yüzdesini tutuyorum  `private final static int ROTTEN_FOOD_ADD_RATE = 10` istenilen orana göre hesaplama yapılıp ona göre ekleniyor. 20 denilirse yüzde 20 olasılıkla ekleyecek örneğin.